style.css

```diff
@@ -796,4 +781,133 @@ input[type='submit']:active {
   * {
     margin: auto;
   }
+
+  .menu {
+    display: none;
+  }
+
+  .desktop-menu {
+    display: flex;
+    align-items: center;
+    gap: 30px;
+  }
+
+  .desktop-menu li a {
+    color: #344563;
+    text-align: center;
+    font-family: Poppins, sans-serif;
+    font-size: 15px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 20px;
+    text-decoration: none !important;
+  }
+
+  .headline-container {
+    background-image: url(assets/images/desktop-header-bg.png);
+  }
+
+  .headline {
+    margin-right: 20%;
+    margin-left: 20%;
+  }
+
+  .social-media-icons {
+    display: inline-flex;
+  }
+
+  .works {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    width: 85%;
+    flex: 1;
+  }
+
+  .work-info {
+    display: flex;
+    align-items: center;
+    padding: 0;
+    gap: 12px;
+    width: auto;
+    margin-left: initial;
+  }
+
+  .language-tags {
+    margin-left: initial;
+  }
+
+  .about-container {
+    width: 100%;
+  }
+
+  .about-myself {
+    padding: 114px 120px;
+    flex-direction: row;
+  }
+
+  .skill-list {
+    width: 100%;
+  }
+
+  .dropdown-content {
+    flex-direction: row;
+  }
+
+  .dropdown-content li {
+    display: block;
+    width: 100px;
+    margin-left: 10px;
+    margin-right: 10px;
+  }
+
+  .top {
+    width: 100%;
+    max-width: 50%;
+    margin-left: 0;
+    margin-right: 0;
+  }
+
+  .top h2,
+  p,
+  .socials {
+    margin-left: 0;
+  }
+
+  .support-text {
+    margin-left: 0;
+    text-transform: uppercase;
+  }
+
+  .resume {
+    margin-left: 0;
+  }
+
+  .languages {
+    width: 100%;
+  }
+
+  hr {
+    height: 2px;
+    width: 100%;
+    background: #dfe1e6;
+    border: none;
+  }
+
+  .contact-form {
+    padding-left: 20%;
+    padding-right: 20%;
+    background-image: url(assets/images/contact-form-bg.svg);
+  }
```

Comment:
- Hey, please, let's control the extra white space here by removing it. see the image below:

![image](https://github.com/NoelLincoln/myportfolio/assets/59397159/bf04a82b-16e6-4be0-a75b-47f15ae09e2f)